### PR TITLE
Help build mac deps using manylinux scripts

### PR DESCRIPTION
**Pull request by ankith26**
A companion PR for #2742

This is my first time fiddling around with sh scripting for docker images, I hope to do some more stuff from the linked PR in this one

@illume do review and verify that all is correct. 

Once these two PRs are done, this branch can be rebased over illume's branch and then merged into main, IG?

### CODE REVIEW

File: buildconfig/macdependencies/install_mac_deps.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+bash libtiff/build-tiff.sh
+bash ogg/build-ogg.sh
+bash flac/build-flac.sh
+bash libwebp/build-webp.sh
+bash freetype/build-freetype.sh
+bash mpg123/build-mpg123.sh
+bash sndfile/build-sndfile.sh
+bash fluidsynth/build-fluidsynth.sh
+bash sdl_libs/build-sdl2-libs.sh
+bash zlib/build-zlib.sh
```
Comment (illume) [inline buildconfig/macdependencies/install_mac_deps.sh:56]: zlib probably needs to go near the top.

Comment (ankith26) [reply buildconfig/macdependencies/install_mac_deps.sh:56]: Nice!

**Comment by ankith26** [comment]
@illume I tried to add libmodplug in the latest commit. Also I added it to the manylinux docker image as well since IIRC you said it is needed on that as well?

**Comment by illume** [comment]
> @illume I tried to add libmodplug in the latest commit. Also I added it to the manylinux docker image as well since IIRC you said it is needed on that?

Thanks heaps! I'll cherry-pick that commit in too.

**Pull request closed without merging**